src/zope/interface/tests/test_advice.py

```diff
@@ -268,7 +249,7 @@ class B(type):
         self.assertEqual(self._callFUT((A, B,)), Meta_B)
 
     @_skip_under_py2
-    def test_multiple_in_hierarchy_py3k(self):
+    def test_multiple_in_hierarchy_py3k(self): # pragma: no cover (tox runs coverage on Python 2)
```

Comment:
Whatdyaknow, you *can* add `@_skip_under_py2` to the .coveragerc and it marks the whole method ignored. I assumed it would only mark that one line. I guess coverage is smarter than I thought it was!

@icemac proposed combining coverage and has an example to do so. I'll try to follow up with a PR for that.